dev-lang/solidity/files/solidity-0.4.24-fix-cmake-external-jsoncpp.diff

```diff
@@ -1,12 +1,13 @@
 diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 24bea3b3..4326927b 100644
+index 0c05208f..2e1ea7f5 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -18,7 +18,9 @@ include(EthCcache)
+@@ -20,8 +20,9 @@ include(EthCcache)
  
  # Let's find our dependencies
  include(EthDependencies)
 -include(jsoncpp)
+-include_directories(SYSTEM ${JSONCPP_INCLUDE_DIR})
 +find_package(PkgConfig REQUIRED)
```

Comment:
should it not depend on `virtual/pkgconfig` then?

Reply:
I guess so, I was looking for a way to do it without pkgconfig in pure cmake but couldn't find any, if you have suggestions I'm open for it, otherwise I'll add the virtual dep

Reply:
it is fine to depend on pkgconfig